backend/src/transactions/club.go

```diff
@@ -7,9 +7,25 @@ import (
 	"github.com/GenerateNU/sac/backend/src/models"
 	"github.com/google/uuid"
 
+	"github.com/gofiber/fiber/v2"
 	"gorm.io/gorm"
 )
 
+func GetAdminIDs(db *gorm.DB, clubID uuid.UUID) ([]uuid.UUID, *errors.Error) {
+	var adminIDs []models.Membership
+
+	if err := db.Where("club_id = ? AND membership_type = ?", clubID, models.MembershipTypeAdmin).Find(&adminIDs).Error; err != nil {
+		return nil, &errors.Error{StatusCode: fiber.StatusInternalServerError, Message: "failed to get admin IDs"}
```

Comment:
ik neither of us are the biggest fan of the current state of errors, but for consistency shouldn't this be moved to the errors package and be `&errors.FailedToGetAdminIDs`